bitnami/mongodb/README.md

```diff
@@ -220,6 +220,7 @@ The following tables lists the configurable parameters of the MongoDB chart and
 | Parameter                                         | Description                                                                                       | Default                        |
 |---------------------------------------------------|---------------------------------------------------------------------------------------------------|--------------------------------|
 | `service.type`                                    | Kubernetes Service type                                                                           | `ClusterIP`                    |
+| `service.name`                                    | MongoDB service name                                                                              | `{mongodb.fullname}-headless`  | 
```

Comment:
Could you rename service.name to service.nameOverride?
We used fullnameOVerridde and nameOverride, in this wat will be the same format.

Reply:
Fixed